deepchem/models/torch_models/tests/test_molformer.py

```diff
@@ -113,3 +113,21 @@ def test_molformer_save_reload(tmpdir):
 
     # all keys values should match
     assert all(matches)
+
+
```

Comment:
Please add a unit test to verify that random weights are loaded instead of pretrained weights when regression and classification models are initialized.

Reply:
Done